src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import static java.lang.Math.PI;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(String.valueOf(color));

```

Comment:
Why are trying to get String representation of string?
```suggestion
        super(color);

```